Add nextcloud maintenance docs (#112)
vshn/appcat-user-docs: docs/modules/ROOT/pages/vshn-managed/nextcloud/maintenance.adoc

= Scheduled Maintenance

By default, there will be a security maintenance on Tuesday night (Central Europe Time).

NOTE: For more information about maintenance procedures, please visit https://products.docs.vshn.ch/products/appcat/nextcloud.html#_recurring_maintenance[the product description].

The day and time can be adjusted as described below. Times are in the CET time zone.

.Nexcloud instance with custom maintenance
[source,yaml]
----
apiVersion: vshn.appcat.vshn.io/v1
kind: VSHNNextcloud
metadata:
  name: nextcloud-app1-prod
  namespace: prod-app
spec:
  parameters:
    maintenance:
      dayOfWeek: wednesday # <1>
      timeOfDay: '23:30:00' # <2>
----
<1> The day when the maintenance should take place
<2> The time when the maintenance should be started

NOTE: In case PostgreSQL database is used then it is also configured to have maintenance by default. See xref:vshn-managed/postgresql/maintenance.adoc[PostgreSQL Maintenance].

== Nextcloud Background Jobs

Nextcloud https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/background_jobs_configuration.html[background jobs] are also part of the maintenance.

> In most cases, the schedule is configured for next hour after the main security update.

Reviewer: ```suggestion
In most cases, the schedule is configured at least one hour after the main security update.
```
